src/Console/Command/FixCommand.php

```diff
@@ -292,8 +294,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : null;
-        if ($stdErr && extension_loaded('xdebug')) {
+        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : $output;
+        if (extension_loaded('xdebug')) {
```

Comment:
I understand the issue, so the RFC on this is when a console does not support multiple streams than only output the result of the reporter? Should be an easy fix